newrelic.js

```diff
@@ -44,6 +44,14 @@ exports.config = {
    * attributes include/exclude lists.
    */
   allow_all_headers: true,
+  application_logging: {
+    /**
+     * When true, logs generated by your application will be forwarded to NRO.
```

Comment:
I'm not sure customers may know what NRO means.  you may want to just put New Relic, or ask one of the tech writers what is the appropriate reference here.